data/successors/successor side missions.txt

```diff
@@ -0,0 +1,488 @@
+			`	As you are making preparations to leave, you realize this would be a good moment to learn about the Predecessors from the experts, especially considering you're liable to be listening to them talk about their new data all the way back.`
+			choice
+				`	(Ask them about the Predecessors.)`
+					goto ask
+				`	(Don't ask them.)`
+					launch
+			label ask
+			`	You approach the group of researchers, still twittering with excitement over the day's work. "I was wondering if you could tell me a little bit about the Predecessors," you say.`
+			`	"Of course!" the head researcher replies, "What dost thou wish to know?"`
+			choice
+				`	"Who were the Predecessors?"`
+					goto who
+				`	"Actually, I didn't have any questions after all."`
+					launch
+			label who
+			`	"Our Predecessors were our now-gone ancestors. Until the time of fifteen hundred years ago, they ruled this space and much beyond as part of their own empire. Many and their bloodlines died as much of the empire fell to ruin, but some made passage through the wormhole to the cluster that we now as their Successors, call our home."`
+			choice
+				`	"What caused the collapse?"`
+					goto collapse
+				`	"How large was their empire?"`
+					goto empire
+				`	"What was their technology like?"`
+					goto tech
+				`	"I have no more questions."`
+					launch
+			label collapse
+			`	The researcher sighs. "We do not know; or better put, the fullness of the truth yet remains obscured. The flesh of space itself was rent, distorted; the effects of this remain today: the holes embedded in the skies, the dying-out of light and even of the hyperlanes. It was a warring time; the Houses turned upon each other or they fled, and space itself was set alight with the blinks of shots and drives---"`
+			`	The researcher stops for a moment, composing themself.`
+			`	"I digress and do apologize. T'is challenging to separate the truth from fiction in this regard. Our Predecessors wielded power far beyond our modern imitations, the sort that sows the seeds of peace or chaos deep within the fabric of the world, the sort of things that poke and prod at places that must remain untouched. 'What caused the collapse?' you ask? Hubris, methinks."`
+			choice
+				`	"How large was their empire?"`
+					goto empire
+				`	"What was their technology like?"`
+					goto tech
+				`	"I have no more questions."`
+					launch
+			label empire
+			`	"The translation device thou wears upon thy person speaks volumes to its scale. Sieasej tells me that at that empire's peak, our scholars had even laid eyes upon thy own world, far to the north! These dead systems that remain where once the core of their empire stood are but a small fraction of what was there. Some, it's sure, now belong to other races, and others, it would seem, have been obscured. We know not how nor why, only that the starcharts of our Predecessors do not mirror those we see today."`
```

Comment:
```suggestion
			`	"The translation device thou wears upon thy person speaks volumes to its scale. Sieasej tells me that at that the empire's peak, our scholars had even laid eyes upon thy own world, so far to the north! These dead systems that remain where once the core of their empire stood are but a small fraction of what was there. Some, it's sure, now belong to other races, and others, it would seem, have been obscured. We know not how nor why, only that the starcharts of our Predecessors do not mirror those we see today."`
```